README.md

```diff
@@ -289,7 +289,7 @@ yo chisel:page "Home" "About Us" "Contact Us" "News"
 When you have the basic setup done, you can start development. To re-compile Twig, SCSS and JavaScript files in real time you can use default task. Type
 
 ```bash
-gulp
+npm run dev
```

Comment:
I'm slightly worried that going into convention without `run` people will be using `npm build` instead of `npm run build` so maybe we will stay with `npm run dev` as official command and keep `npm start` unofficial.